sim/shaman/unleash_elements.go

```diff
@@ -142,7 +142,7 @@ func (shaman *Shaman) registerUnleashLife() {
 		},
 		OnExpire: func(aura *core.Aura, sim *core.Simulation) {
 			for _, spell := range affectedSpells {
-				spell.CostMultiplier -= 0.2
+				spell.DamageMultiplierAdditive -= 0.2
```

Comment:
Just double checking that this was intentional?

Reply:
Yes, because the `OnGain` was never reverted in the `OnExpire`. I double checked the talent.